ext/opentelemetry-ext-requests/tests/test_requests_integration.py

```diff
@@ -36,6 +37,7 @@ def setUp(self):
         httpretty.register_uri(
             httpretty.GET, self.URL, body="Hello!",
         )
+        httpretty.register_uri(httpretty.POST, self.URL, body="World!")
```

Comment:
Nit: Do you really need to create a new one?

Reply:
I actually added this to test the collection of the request body, but I guess it's not really mandatory for the test.
Removed.